tests/templates/kuttl/upgrade/00-install-zk.yaml.j2

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: zookeeper.stackable.tech/v1alpha1
+kind: ZookeeperCluster
+metadata:
+  name: test-zk
+spec:
+  servers:
+    roleGroups:
+      default:
+        replicas: 1
+        config:
+          myidOffset: 10
```

Comment:
This is not required for a 1 role group cluster.

Reply:
done